modules/user-facing-text/main/index.coffee

```diff
@@ -0,0 +1,45 @@
+_ =
+  localisedText: {}
+
+userFacingTextEmitter = new hx.EventEmitter
+
+userFacingText = (stringPath, locales) ->
+  if arguments.length
+    strings = if hx.isString(stringPath)
+      [stringPath]
+    else stringPath
+
+    if locales?
+      newText = tmp = {}
+      for string, i in strings
+        if i is strings.length - 1
+          # Merge the locales to prevent mutation
+          tmp[string] = hx.merge locales
+        else
+          tmp[string] = {}
+          tmp = tmp[string]
+      userFacingTextEmitter.emit('change')
+      _.localisedText = hx.merge.defined(_.localisedText, newText)
+      return
+    else
+      textObject = undefined
+      tmp = _.localisedText
+      for string, i in strings
+        if i is strings.length - 1
+          textObject = tmp[string]
+        else if tmp and tmp[string]
+          tmp = tmp[string]
+        else break
+
+      if not textObject
+        hx.consoleWarning 'hx.userFacingText.localisedText: No text was found for locale for the path ' + stringPath
+      else
+        textObject[hx.preferences.locale()] || textObject['en'] # There must be a fallback in english.
```

Comment:
Is it worth checking if the `hx.preferences.locale()` version exists then checking for the `en` version and then showing a warning if it doesn't exist or should there _always_ be an english version?
I guess the real question is 'Should people be configuring their own non-hexagon text with this module or is it just for configuring hexagon?'